pkg/repository/prisma/job_run.go

```diff
@@ -130,14 +139,7 @@ func (j *jobRunEngineRepository) GetJobRunsByWorkflowRunId(ctx context.Context,
 	})
 }
 
-func setJobRunStatusRunning(ctx context.Context, pool *pgxpool.Pool, queries *dbsqlc.Queries, l *zerolog.Logger, tenantId, jobRunId string) (*pgtype.UUID, error) {
-	tx, err := pool.Begin(ctx)
-
-	if err != nil {
-		return nil, err
-	}
-
-	defer sqlchelpers.DeferRollback(context.Background(), l, tx.Rollback)
+func setJobRunStatusRunning(ctx context.Context, tx dbsqlc.DBTX, queries *dbsqlc.Queries, l *zerolog.Logger, tenantId, jobRunId string) (*pgtype.UUID, error) {
```

Comment:
```suggestion
func (s *sharedRepository) setJobRunStatusRunning(ctx context.Context, tx dbsqlc.DBTX, tenantId, jobRunId string) (*pgtype.UUID, error)
```

(along with corresponding method changes)